presto-docs/src/main/sphinx/admin/properties-node-scheduler.rst

```diff
@@ -76,12 +76,22 @@ For example, setting ``region,rack,machine`` means a network location contains t
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 * **Type:** ``string``
-* **Allowed values:** ``flat``, ``file``
+* **Allowed values:** ``flat``, ``file``, ``subnet``
 * **Default value:** ``flat``
 
-Sets the network topology type. To use this option, ``node-scheduler.policy`` must be set to
-``topology``. ``flat`` has only one segment, with one value for each machine.
-``file`` loads the topology from a file as described below.
+Sets the network topology type. To use this option, ``node-scheduler.policy``
+must be set to ``topology``.
+
+``flat`` : the topology has only one segment, with one value for each machine.
+
+``file``: the topology is loaded from a file using
```

Comment:
This is fine, but we could simplify it a bit if we add new sections. Then we don't need to mention properties here, or mention the `type` in the properties descriptions below.
```
Network topology
----------------

...

``file`` network topology
-------------------------

...

``subnet`` network topology
---------------------------

...
```

Reply:
If we do that we have to make sure to use the correct level underline for the titles.. probably """""""

Reply:
Thanks @electrum, I changed it so that titles use `^^^^^` and properties under them use `""""`. It looks good to me, @mosabua wdyt?

<img width="966" alt="Screen Shot 2020-11-23 at 5 45 53 PM" src="https://user-images.githubusercontent.com/6829746/100036634-6d3d5a80-2db5-11eb-8267-9819e224dddb.png">